README.md

```diff
@@ -18,7 +22,7 @@ The Jamulus website is translated using [Weblate](https://hosted.weblate.org/eng
 
 To edit an individual file, you can use the Github web interface or make a fork of the Jamulus repository (button at [top right here](https://github.com/jamulussoftware/jamuluswebsite)). In your fork, you can edit multiple website pages (or of course use git on the command line if you prefer). When you are done, create a pull request that we will review, translate and publish in the next release.
 
-To view the website offline one your own machine, you will need to clone it locally and use [Jekyll](https://jekyllrb.com/) to build it. Please ask on [in the discussions](https://github.com/jamulussoftware/jamulus/discussions) for help with this if necessary.
+To view the website offline on your own machine, you will need to clone it locally and use [Jekyll](https://jekyllrb.com/) to build it. Please ask on [the discussions](https://github.com/jamulussoftware/jamulus/discussions) for help with this if necessary.
```

Comment:
Link the release process Site here. There's a guide on how to do that in the linkcheck section

Reply:
I don't think "... in the discussions" reads well.
```suggestion
To view the website offline on your own machine, you will need to clone it locally and use [Jekyll](https://jekyllrb.com/) to build it. Please ask in [Discussions](https://github.com/jamulussoftware/jamulus/discussions) for help with this if necessary.
```

Reply:
```suggestion
To view the website offline on your own machine, you will need to clone it locally and use [Jekyll](https://jekyllrb.com/) to build it. Please ask on [the discussions](https://github.com/jamulussoftware/jamulus/discussions) for help with this is necessary.
```

Reply:
Eh?  "...for help with is necessary"?  That's not right.

Reply:
> Link the release process Site here. There's a guide on how to do that in the linkcheck section

Hm, there's a bit more to it than what the instructions say - you need to install po4a too and that's Linux only. Maybe point them to the artifact produced by the GH actions?

Reply:
Hmm. Ok. Then we need to add that the po4a deb needs to be installed on the release process site.

Reply:
I generally use the artifact - it's easier :).

Reply:
I've made it into a small section of its own and just limited the instructions to using the artifact. I think that getting into instructions on installing po4a and running the script makes it over-complicated.